index.html

```diff
@@ -151,29 +151,31 @@ <h2>
     </section>
     <section>
       <h2>
-        Interface definitions
-      </h2>
-      <section>
-        <h2>
-          Extensions to the <a>Screen</a> interface
-        </h2>The CSSOM View specification defines a <code>Screen</code>
-        interface [[!CSSOM-VIEW]], which this specification extends:
-        <pre class='idl'>
+        Extensions to the <a>Screen</a> interface
+      </h2>The <a data-cite="CSSOM-View">CSSOM View Module</a> specification
+      defines a <code>Screen</code> interface, which this specification
```

Comment:
I think here we should to formally define and link to `Screen`. 

```suggestion
      defines the <code><dfn data-cite="CSSOM-View#screen">Screen</dfn></code> interface, which this specification
```